en/identity-server/7.1.0/docs/apis/restapis/idp.yaml

```diff
@@ -468,6 +478,11 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/IdentityProviderResponse'
+              examples:
+                identityProviderWithSystemDefinedAuthenticator:
+                  $ref: '#/components/examples/systemDefinedIdentityProviderResponse'
```

Comment:
Shall we follow a consistent patter in defining this sample object.
In above it's 'IdentityProviderWithSystemDefinedAuthenticatorPOSTExample' and here it's 'systemDefinedIdentityProviderResponse'